apps/web/src/components/PaginatedTable/components/RollupFilter.tsx

```diff
@@ -0,0 +1,53 @@
+import type { FC } from "react";
+
+import { Dropdown } from "~/components/Dropdown/Dropdown";
+import type { DropdownProps } from "~/components/Dropdown/Dropdown";
+import { RollupIcon } from "~/components/RollupIcon";
+import type { Rollup } from "~/types";
+
+const rollups: Rollup[] = [
```

Comment:
I'd rather use the existing Rollups enum instead of redefining the list of rollups here.

Reply:
+1

Reply:
[Done](https://github.com/Blobscan/blobscan/pull/496/commits/3444ceb62a8939b49946019a41f8b8d761168fb0) by @luis-herasme in